java-ws/server/Todo/src/main/java/com/couchbase/lite/todo/model/Task.java

```diff
@@ -198,6 +212,9 @@ public static Blob getPhoto(UserContext context, String taskListId, String taskI
 
         Blob blob = doc.getBlob(KEY_IMAGE);
         if (blob == null) { throw ResponseException.NOT_FOUND("image"); }
+
+        System.out.println("Blob image to JSON (when get image): "+ blob.toJSON());
```

Comment:
maybe this should be "when delete image"?

Reply:
This is in getPhoto() method

Reply:
This is in method getPhoto()

Reply:
I see... got it.